lib/skate_web/components/core_components.ex

```diff
@@ -0,0 +1,13 @@
+defmodule SkateWeb.CoreComponents do
+  @moduledoc """
+  Provides core UI components.
+  """
+
+  def cdn_route(conn, path) do
+    {static_href_module, static_href_fn} = Application.get_env(:skate, :static_href)
+    static_href = fn conn, path -> apply(static_href_module, static_href_fn, [conn, path]) end
+
+    static_href.(conn, path)
+    "https://localhost:4000#{static_href.(conn, path)}"
```

Comment:
Looks like some leftover local testing code?

Reply:
Yep! It works on my machine, and now dev-blue doesn't load :)